pkg/tso/keyspace_group_manager.go

```diff
@@ -730,6 +735,10 @@ func (kgm *KeyspaceGroupManager) updateKeyspaceGroup(group *endpoint.KeyspaceGro
 	}
 	kgm.kgs[group.ID] = group
 	kgm.ams[group.ID] = am
+	// If the group is the split target, add it to the splitting group map.
+	if group.IsSplitTarget() {
+		kgm.splittingGroups[group.ID] = struct{}{}
```

Comment:
What if it transfers the leader? Will splittingGroups become invalidated?

Reply:
`groupSplitPatroller` will check if the current allocator is the leader.